temporal-testing/build.gradle

```diff
@@ -1,11 +1,30 @@
+plugins {
+    id 'application'
+}
+
 description = '''Temporal Workflow Java SDK testing'''
 
 dependencies {
+    api group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
     api project(':temporal-sdk')
 
     implementation group: 'com.google.guava', name: 'guava', version: '30.1.1-jre'
     implementation group: 'com.cronutils', name: 'cron-utils', version: '9.1.5'
 
     testImplementation group: 'junit', name: 'junit', version: '4.13.2'
     testImplementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
-}
\ No newline at end of file
+}
+
+task testServiceServer(type: CreateStartScripts) {
+    mainClassName = 'io.temporal.internal.testservice.TestServiceServer'
+    applicationName = 'test-service-server'
+    outputDir = new File(project.buildDir, 'tmp')
+    classpath = startScripts.classpath
+    dependsOn 'sourcesJar'
+}
+
+// Run: ./gradlew installDist :temporal-testing:testServiceServer
+applicationDistribution.into('bin') {
```

Comment:
I would also like to publish with maven, if that's okay with y'all.  It can be a separate PR; I'd like to get this checked in to avoid a third round of merge conflicts.

Notes:
* The generated script is useful for debugging or if you don't want to run a container.
* In production, we're using a container with only the .jar files in `./temporal-testing/build/install/temporal-testing/lib`.  
* We just uploaded them to our artifactory and want to do something more official.

I don't know anything about maven or java conventions so I'm not sure what you'd want and would need guidance.